src/_pytest/terminal.py

```diff
@@ -154,6 +154,14 @@ def pytest_addoption(parser: Parser) -> None:
         dest="no_summary",
         help="Disable summary",
     )
+    group._addoption(
+        "--fold-skipped",
```

Comment:
if we don't plan to to add more variants, lets go with `--no-fold-skipped` as a store false, default true action following  the way flags work in tools like click

Reply:
Done.